docs/topics/mta-web-assessment-module.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+:_content-type: REFERENCE
+[id="mta-web-assessment-module_{context}"]
+= Assessment Module
+
+The assessment process for applications that are to be migrated in {ProductFullName} has been separated from other tool features.
+
```

Comment:
Nothing is migrated in MTA. "The assessment process for applications has been separated from other tool features." [Is this true?]